packages/components/RoadmapSection.tsx

```diff
@@ -1,92 +1,90 @@
 import { LocalizedText, type TranslationToken } from "@packages/features/i18n-context";
+import { classes } from "@packages/utils/classes";
 
 import Section from "./Section";
 
-export default function RoadmapSection() {
-  const bgColor = useColorModeValue("bg-gray-100", "bg-gray-900");
-  return (
-    <Section className={bgColor}>
-      <Flex justifyContent="space-between" py="1rem">
-        <Heading size="md" fontWeight={600} textAlign="left">
-          <LocalizedText token="roadmap.title" />
-        </Heading>
-        <Text textAlign="right">
-          <Link href={roadMapsLinks.all} isExternal>
-            <LocalizedText token="roadmap.see-all" />
-          </Link>
-        </Text>
-      </Flex>
-
-      <Grid
-        gridTemplateColumns={{
-          base: "1fr",
-          sm: "repeat(2, 1fr)",
-          lg: "repeat(4, 1fr)",
-        }}
-        gap="2rem"
-        my="2rem"
-      >
-        {cardList.map((cardProps) => (
-          <CardItem key={cardProps.title} color={cardProps.color} link={cardProps.link} title={cardProps.title} />
-        ))}
-      </Grid>
-    </Section>
-  );
-}
-
 interface CardItemProps {
   title: TranslationToken;
   link: string;
-  color: string;
-}
-
-function CardItem({ title, link, color }: CardItemProps) {
-  return (
-    <Link href={link} isExternal _hover={{ textDecoration: "none" }}>
-      <Flex
-        py="2rem"
-        transition="box-shadow 300ms ease-in-out"
-        border="2px solid"
-        borderColor={color}
-        borderRadius="10px"
-        justifyContent="center"
-        _hover={{
-          boxShadow: `-8px 8px 0 ${color}`,
-        }}
-      >
-        <Heading size="md" fontWeight={400} textDecoration="none">
-          <LocalizedText token={title} />
-        </Heading>
-      </Flex>
-    </Link>
-  );
+  border: string;
+  shadow: string;
 }
 
 const cardList: CardItemProps[] = [
   {
     title: "roadmap.web-programming",
     link: roadMapsLinks.webProgramming,
-    color: "#17A9BC",
+    border: "border-[#17A9BC]",
+    shadow: "hover:shadow-[#17A9BC_-8px_8px_0px]",
   },
   {
     title: "roadmap.ux-design",
     link: roadMapsLinks.uxDesign,
-    color: "#F99223",
+    border: "border-[#F99223]",
+    shadow: "hover:shadow-[#F99223_-8px_8px_0px]",
   },
   {
     title: "roadmap.react",
     link: roadMapsLinks.react,
-    color: "#FF4CFF",
+    border: "border-[#FF4CFF]",
+    shadow: "hover:shadow-[#FF4CFF_-8px_8px_0px]",
   },
   {
     title: "roadmap.data",
     link: roadMapsLinks.introToData,
-    color: "#b794f4",
+    border: "border-[#b794f4]",
+    shadow: "hover:shadow-[#b794f4_-8px_8px_0px]",
   },
 ];
+
+function CardItem({ title, link, border, shadow }: Readonly<CardItemProps>) {
+  return (
+    <a href={link} target="_blank" rel="noreferrer">
+      <div
+        className={classes(
+          "flex justify-center py-8 border-2 rounded-lg transition-shadow duration-300 ease-in-out",
+          border,
+          shadow,
```

Comment:
Can we do something like this?

```suggestion
          "hover:shadow-[8px_8px_0px]",
          color,
```